tx-parser/tests/parser.rs

```diff
@@ -26,7 +26,7 @@ fn test_entities() {
     let input = r#"[[:db/add 101 :test/a "v"]
 [:db/retract 102 :test/b "w"]]"#;
 
-    let edn = parse::value(input).unwrap();
+    let edn = parse::value(input).unwrap().fold();
```

Comment:
I know I suggested `into_inner`, but maybe we should be explicit and say `without_spans`?  That'll certainly read easiest...